### PR TITLE
Text on rust-keylime and more body on what the project achieves.

### PR DESCRIPTION
**Pull request by lukehinds**
This change explains the role of keylimes main components and more on what the project plans to achieve, it also explains the current status of the rust-keylime agent code base.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,66 @@
...
+users alike, without the need for a deep understanding of the lower levels of a
+TPM's operations. Amongst many scenarios, it well suited to tenants who need to
+remotely attest machines not under their own full control (such as a consumer of
+hybrid cloud.)
+
+Keylime can be driven with a CLI application, web front end and a set of
+RESTful APIs.
+
+Keylime consists of three main components; The verifier, registrar and the
+agent. The verifier computes the measurements of the cryptographic TPM
```
Comment (jetwhiz) [inline README.md]: This sentence might be a bit jargon-y for the first few paragraphs of the README: "The verifier computes the measurements of the cryptographic TPM quotes accessed by the agents communication with the hosts TPM chip."

Maybe we can make this something more along the lines of "The verifier continuously verifies the integrity state of the machine that the agent is running on." ?

File: README.md [hunk 2]
```diff
@@ -1,21 +1,66 @@
...
+Lastly is the agent, which is deployed to the remote machine that is to be
+measured or provisoned with secrets stored within an encrypted payload released
+by the TPM.
+
+### Rust based Keylime Agent
+
+The verifier, registrar and agent are all developed in Python and situated
+in this repository `keylime`. The agent is currently undergoing a port to the
+[Rust programming language](https://www.rust-lang.org), with this work taking
+place in the [keylime-rust repository](https://github.com/keylime/rust-keylime).
```
Comment (jetwhiz) [inline README.md]: Should the link text here be `rust-keylime repository` to match the repository name?

**Comment by jetwhiz** [comment]
Looks good!  Just a couple comments/questions/changes (see above).

**Comment by lukehinds** [comment]
> Looks good to me. I prefer commit messages follow [this guideline](https://chris.beams.io/posts/git-commit/), but it looks like the project has not been enforcing that.

I will include this in the contributors text, always nice to have these things standardised!

File: README.md (hunk 2 above)
Comment (lukehinds) [reply README.md]: Good capture!

File: README.md (hunk 1 above)
Comment (lukehinds) [reply README.md]: Will use you change, sounds good to me.

**Pull request merged**